Lib/fontTools/feaLib/parser.py

```diff
@@ -10,13 +10,15 @@
 import re
 
 
+
 log = logging.getLogger(__name__)
 
 
 class Parser(object):
 
     extensions = {}
     ast = ast
+    ignore_comments=True
```

Comment:
when the equal sign is used in assignments, it usually is surrounded by spaces
(to be clear, I don't enjoy playing the pep8-er).